larpandora/LArPandoraEventBuilding/LArPandoraShower/Tools/ShowerPCADirection_tool.cc

```diff
@@ -156,9 +152,10 @@ namespace ShowerRecoTools {
     }
 
     //Otherwise Check against the RMS of the shower. Method adapated from EMShower Thanks Mike.
-    double RMSGradient = RMSShowerGradient(spacePoints_pfp,ShowerCentre,PCADirection);
+    double RMSGradient = IShowerTool::GetLArPandoraShowerAlg().RMSShowerGradient(spacePoints_pfp,ShowerCentre,PCADirection, fNSegments);
 
-    if(RMSGradient < 0){
+    // If the alg fails to calculate the gradient it will return 0. In this case do nothing
+    if(RMSGradient < -std::numeric_limits<double>::epsilon()){
```

Comment:
This conditional appears to be covering more than is described in the comment. Does the comment need updating or the code?

Reply:
I did add a further comment here, but it is not showing for some reason: https://github.com/PandoraPFA/larpandora/pull/18/commits/092f7a8ba52e100d1ecb9ab54bf79bf81e7cb752#diff-73a4d6bdd95883109001cb88d814c8a0757eca1a39da74fc1020e9305ba55fc7R158

Reply:
And now I realised there was a typo in the comment, fixed here: https://github.com/PandoraPFA/larpandora/pull/18/commits/ec06d6ebfded4f170970e03abc826d50cd46cdb3